modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -854,6 +854,95 @@ definition {
 		}
 	}
 
+	@description = "LPS-175201 -Verify if is possible create a validation using the oldValue function with the Text field using Commerce Order object"
+	@priority = 5
+	test CanCreateValidationUsingOldValueFunctionInSystemObject {
+		task ("Given: an account is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Accounts Name",
+				accountEntryType = "Business");
+		}
+
+		task ("And given: a custom validation using the oldValue function is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Postal Address");
+
+			ObjectAdmin.goToValidationsTab();
+
+			ObjectCustomValidation.addObjectValidation(
+				validationName = "Custom Validation",
+				validationType = "Expression Builder");
+
+			ObjectCustomValidation.gotoCustomValidation(validationName = "Custom Validation");
+
+			ObjectCustomValidation.gotoTab(tabName = "Conditions");
+
+			ObjectCustomValidation.insertScriptOnValidation(scriptValidation = "oldValue('Name') == 'Test'");
+
+			ObjectCustomValidation.addErrorMessage(errorMessage = "Invalid Value");
+
+			PortletEntry.save();
+		}
+
+		task ("And given: an address is created to the account") {
+			Account.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Accounts Name");
+
+			Account.gotoAssigneeTab(assigneeTab = "Addresses");
+
+			Account.addAddress(
+				addressCity = "Diamond Bar",
+				addressCountry = "United States",
+				addressName = "Name Test",
```

Comment:
Hey @victorhcunha, the variable addressName must have the same value as the expression. For example, if you have to know if the "Name" field will have the old value "Test", then the addressName must be something Like: `addressName = "Test"` .